proposals/sar_backscatter.json

```diff
@@ -89,15 +89,6 @@
                 "type": "boolean"
             }
         },
-        {
-            "name": "noise_power",
```

Comment:
It's a target requirement. So not critical. Still we have added all the other target requirements. Here we don't know what this item means. So we removed it for now. 
So we'll have a incomplete list of the target requirements.

Reply:
On the other hand, CARD4L has no direct connection to sar_backscatter. It's only really relevant to the ard_* processes.

Reply:
**mask:** create manually via no-data and data
**contributing_area**: double check what it means. required.
**noise_power**: double check what it means. could be left out and added later
**date**: find better naming

 **rtc**: radiometric correction currently fixed to the method chosen by the backend and reported to metadata.

Reply:
I did the renaming from incidence_angle to local_incidence_angle and from date to date_offset. I'll also apply the changes in STAC.

Reply:
@przell Is this all resolved?